Anshul/ESPN.PY

```diff
@@ -0,0 +1,63 @@
+import time
+
+from selenium import webdriver
+
+import unittest
+
+from selenium.webdriver import ActionChains
+from selenium.webdriver.common.by import By
+
+
+class ESPN(unittest.TestCase):
+    def setUp(self) -> None:
+        global IPL1
+        IPL1=webdriver.Chrome()
+        IPL1.get("https://www.espncricinfo.com")
+        IPL1.maximize_window()
+
+
+    def test_OutPlyerNmae(self):
+        KeySeriessexction=IPL1.find_element(By.XPATH,"//span[text()='World Cup Super League']")
+        Ac=ActionChains(IPL1)
+        Ac.move_to_element(KeySeriessexction).perform()
+        IPL2023=IPL1.find_element(By.XPATH,"//span[text()='IPL 2023']/parent::a")
+        IPL2023.click()
+        time.sleep(20)
+        yesbutton=IPL1.find_element(By.ID,"wzrk-confirm")
+        yesbutton.click()
+        time.sleep(10)
+        Result=IPL1.find_element(By.XPATH,"//span[text()='Results']")
+        Ac=ActionChains(IPL1)
+        Ac.move_to_element(Result).click().perform()
+        Lastmatchwin=IPL1.find_element(By.XPATH,"//div[@class='ds-block']//a")
+        Lastmatchwin.click()
+        time.sleep(13)
+        #IPLMenu=IPL1.find_element(By.XPATH,"//span[text() = 'Indian Premier League 2023']/../ i")
+        #IPLMenu.click()
+        # ipl2021=IPL1.find_element(By.XPATH,"//li[@title='Indian Premier League 2023']")
+        # Ac1=ActionChains(IPL1)
+        # Ac1.move_to_element(ipl2021).click().perform()
+        lastIpl=IPL1.find_element(By.XPATH,"//div[@class='ds-w-[288px] card scorecard']")
```

Comment:
The class name contains white spaces and looks volatile.
Can we use contains in xpath to make it more generic